src/sass/_forms.scss

```diff
@@ -148,6 +159,17 @@
   }
 }
 
+.cloud-init {
+  .override-form {
+    align-items: flex-end;
+    flex-direction: column-reverse;
+
+    > :first-child {
+      width: 100%;
+    }
+  }
+}
+
```

Comment:
Is this supposed to do anything about the width of the YAML editor being too small on medium screens? Because I still see it as small as it was before:

![image](https://github.com/canonical/lxd-ui/assets/56583786/2672ef54-cdc2-41f0-94fe-518cae98d008)

Reply:
On the demo server it looks all right, did you maybe need to pull?

Reply:
Right, apparently I pulled before the latest update and missed it.